docs/source/format/Integration.rst

```diff
@@ -328,6 +328,25 @@ Null: ::
       "name": "null"
     }
 
```

Comment:
@nealrichardson Can you look at the wording here?

Reply:
LGTM. The example is great, thanks for adding it.